openage/__main__.py

```diff
@@ -77,6 +77,10 @@ def main(argv=None):
                             help=("upon throwing an exception a debug break is "
                                   "triggered. this will crash openage if no "
                                   "debugger is present"))
+    if sys.platform == 'win32':
+        global_cli.add_argument(
+            "--add-dll-search-path", action='append', dest='dll_paths',
```

Comment:
this is not used yet?
and i'd favor naming it `dll-search-path` without `add`, and the `dest` can then be the same. If needed, we could also add `;`-splitting for multiple values (and name it plural `paths` then)